view/frontend/web/template/payment/offsite-internetbanking-form.html

```diff
@@ -116,8 +114,7 @@
                                 <img class="bbl" />
                             </div>
                             <div class="omise-banking-text-wrapper">
-                                <span class="title">Bangkok Bank</span><br/>
-                                <span class="rate secondary-text">Fee: 10 THB (same zone), 20 THB (out zone)</span>
```

Comment:
@jonrandy Can you also remove `.secondary-text` out from its stylesheet? (note sure about `.rate`)